src/main/java/uk/gov/hmcts/divorce/common/model/State.java

```diff
@@ -160,13 +160,23 @@
     )
     ConditionalOrderComplete("ConditionalOrderComplete"),
 
+    @CCD(
+        name = "AOS Drafted",
+        label = "# **${[CASE_REFERENCE]}** ${applicant1LastName} **&** ${applicant2LastName}\n### **${[STATE]}**\n",
+        access = {CaseAccessAdministrator.class}
+    )
+    AOSDrafted("AOSDrafted"),
```

Comment:
We probably should be consistent and have lowercase "os", maybe should be AosDrafted("AosDrafted"),

Reply:
done